NetCoreRestApi/DataLayer/Entities/ProductEntity.cs

```diff
@@ -2,11 +2,9 @@
 
 namespace DataLayer.Entities
 {
-    public class ProductEntity : BaseEntity
-    {        
-        public string Name { get; set; }
-        public string Description { get; set; }
-        public ICollection<CategoryEntity> Categories { get; set; }
+    public class ProductEntity<T> : ItemEntity<T>
```

Comment:
Fix to class ProductEntity : ItemEntity<int>

Reply:
done